NAMESPACE

```diff
@@ -55,6 +57,7 @@ export(inactivateWorkflowTasks)
 export(lastPathElement)
 export(loadSimulationWithUpdatedPaths)
 export(loadStudyDesign)
+export(loadUserDefinedTask)

```

Comment:
should we call it `addUserDefinedTask `instead of `loadUserDefinedTask`?
When I see `loadUserDefinedTask` I think it would override the (only one) user defined task - which is not the case anymore (now that we have the list of userdef tasks)